config/main.py

```diff
@@ -788,6 +798,70 @@ def del_vlan_member(ctx, vid, interface_name):
     db.set_entry('VLAN', vlan_name, vlan)
     db.set_entry('VLAN_MEMBER', (vlan_name, interface_name), None)
 
+@vlan.group('dhcp_servers')
```

Comment:
We need to unify nomenclature. Currently, we refer to the DHCP IP Helper addresses as "DHCP servers" in ConfigDB. However, the IPs are not _necessarily_ IPs of actual servers, but possibly the broadcast address of another subnet which contains a DHCP server, which is why they are commonly known as "DHCP IP Helper Addresses" or "DHCP Helper Addresses". The latter is what is used in the `show vlan brief` output. Throughout this PR, the naming also changes. We need to choose one and stick with it for the CLI (both for the command names as well as all output). What is your opinion: "DHCP Servers", "DHCP IP Helpers" or "DHCP Helpers"?